token/scripts/localnet.sh

```diff
@@ -48,6 +48,7 @@ pnpm exec hardhat initializeIndex \
   --contract $CONTRACT_ZETACHAIN \
   --tokens "$TOKEN_LIST" \
   --ratios 50,30,20 \
+  --price-symbols "mBTC,mETH,mDOGE" \
```

Comment:
these should be the real mainnet symbols? we want to display mainnet prices even on testnet and for the mock tokens.